### PR TITLE
Profile argument

### PR DESCRIPTION
**Pull request by friedjoff**

### CODE REVIEW

**Comment by friedjoff** [comment]
@danji90 please update data transformation to automatically generate "profiled" `types.json` which is currently modified manually. Also replace slope test example with some real-world data. Thx!

Regarding data merge strategy: I guess we should only implement that when needed.

File: lib/data/sql/V5__export_projections.sql [hunk 1]
```diff
@@ -192,7 +192,51 @@ COPY (
 ----- types
 
 COPY
-    (WITH additional AS
+    (SELECT jsonb_build_object(
+      'lu', (SELECT jsonb_build_object('standortTypen', (SELECT json_agg(jsonb_build_object('stoNr', sto_nr,
```
Comment (friedjoff) [inline lib/data/sql/V5__export_projections.sql:196]: JSON keys should be in english, similiar to other types.

File: lib/test/info.test.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ test('valid german translation', () => {
   expect(info('treeType', 100).de).toStrictEqual('Tanne');
 });
 
+test('valid profile-specific german translation', () => {
+  // TODO: replace with real-life data
+  expect(info('slope', '<20', 'lu').de).toStrictEqual('kleiner 20 Prozent');
```
Comment (friedjoff) [inline lib/test/info.test.js:9]: Please replace this with some real LU data.

File: lib/data/sql/V1.1__import_lu.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE lu_standorttypen (STO_Nr TEXT, STO_DEU TEXT, STO_LAT TEXT, EIGNUNG TEXT, WB_VERJ_ENT TEXT, WB_PFL TEXT, BESCHREIBUNG TEXT, HOEHENVERBREITUNG TEXT, VEGETATION TEXT, VORWALDBAUMARTEN TEXT, GESGR_Nr TEXT, VERDRISK TEXT, PRIORITAET TEXT);
```
Comment (friedjoff) [inline lib/data/sql/V1.1__import_lu.sql:1]: What do you think about using Postgres schemas for each profile?

File: lib/data/sql/V1.1__import_lu.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+COPY lu_standorttypen
+FROM '/data/profiles/lu/lu_standorttypen.csv'
```
Comment (friedjoff) [inline lib/data/sql/V1.1__import_lu.sql:3]:
```suggestion
FROM '/data/profiles/lu/standorttypen.csv'
```

The prefix is a bit redundant.

**Review by friedjoff — COMMENTED** [review]
Changes to JSON files beside `types.json` should be ignored (reason is propably missing spatial data). Missing spatial data might also affect `types.json` in a negative way. This might also fix some unit tests.

We could also think about extracting the generation of `types.json` in a separate migration file, to make the code a bit more maintainable.

File: lib/data/sql/V5__export_projections.sql (hunk 1 above)
Comment (danji90) [reply lib/data/sql/V5__export_projections.sql:196]: Done. Check out my English definitions and see if they are comprehensible.

File: lib/test/info.test.js (hunk 2 above)
Comment (danji90) [reply lib/test/info.test.js]: Done

File: lib/data/sql/V1.1__import_lu.sql (hunk 4 above)
Comment (danji90) [reply lib/data/sql/V1.1__import_lu.sql]: Removed

File: lib/data/sql/V1.1__import_lu.sql (hunk 3 above)
Comment (danji90) [reply lib/data/sql/V1.1__import_lu.sql:1]: Maybe at a later time, but we can put it in the backlog for now

**Pull request merged**